plugins/modules/azure_rm_aks.py

```diff
@@ -1387,6 +1450,35 @@ def create_addon_profile_instance(self, addon):
         return result
 
 
+    # AKS only supports a single UserAssigned Identity
+    def update_identity(self, param_identity, curr_identity):
+        user_identity = None
+        changed = False
+        current_managed_type = curr_identity.get('type', 'SystemAssigned')
+        current_managed_identity = curr_identity.get('user_assigned_identities', {})
+        param_managed_identity = param_identity.get('user_assigned_identities')
+
+        # If type set to SystamAssigned, and Resource has SystamAssigned, nothing to do
+        if 'SystemAssigned' in param_identity.get('type') and current_managed_type == 'SystemAssigned':
+            pass
+        # If type set to SystemAssigned, and Resource has current identity, remove UserAssigned identity
+        elif param_identity.get('type') == 'SystemAssigned':
+            changed = True
+        # If type in module args contains 'UserAssigned'
+        elif 'UserAssigned' in param_identity.get('type'):
+            if param_managed_identity not in current_managed_identity.keys():
+                user_identity = { param_managed_identity: {} }
```

Comment:
whitespace after '{,}'
```suggestion
                user_identity = {param_managed_identity: {}}
```